sentinelhub/data_collections.py

```diff
@@ -372,6 +375,8 @@ def define(cls, name, *, api_id=None, catalog_id=None, wfs_id=None, service_url=
         :type collection_id: str or None
         :param is_timeless: `True` if a data collection can be filtered by time dimension and `False` otherwise
         :type is_timeless: bool
+        :param dem_instance: DEM instance for DEM collections (one of MAPZEN, COPERNICUS_30, COPERNICUS_90)
```

Comment:
Instead of listing the options, you can perhaps reference the documentation:
```
one of the options listed in `DEM documentation <https://docs.sentinel-hub.com/api/latest/data/dem/#deminstance>`__
```